File: library/src/main/java/com/twiceyuan/imagepickcompat/options/CropOptions.java
package com.twiceyuan.imagepickcompat.options;

/**
 * Created by twiceYuan on 2017/6/19.
 * <p>
 * Options about crop action.
 */
public class CropOptions {

    public int     outputX;
    public int     outputY;
    public int     aspectX;
    public int     aspectY;
    public boolean scale;

    public CropOptions() {
        outputX = 300;
        outputY = 300;
        aspectX = 1;
        aspectY = 1;
        scale = true;
    }
}

File: library/src/main/java/com/twiceyuan/imagepickcompat/options/CropOptions.kt
package com.twiceyuan.imagepickcompat.options

/**
 * Created by twiceYuan on 2017/6/19.
 *
 *
 * Options about crop action.
 */
data class CropOptions(
    var outputX: Int? = null,
    var outputY: Int? = null,
    var aspectX: Int = 1,
    var aspectY: Int = 1,
    var scale: Boolean = true,
)
